packages/react-instantsearch/examples/autocomplete/package.json

```diff
@@ -2,18 +2,19 @@
   "private": true,
   "devDependencies": {
     "identity-obj-proxy": "3.0.0",
+    "jest-environment-jsdom-11.0.0": "^20.0.9",
```

Comment:
this is now an external package is that it?

Reply:
http://facebook.github.io/jest/docs/en/configuration.html#testenvironment-string